cmd/osmosisd/cmd/genesis.go

```diff
@@ -0,0 +1,152 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	claimtypes "github.com/c-osmosis/osmosis/x/claim/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/server"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+	"github.com/spf13/cobra"
+	appparams "github.com/c-osmosis/osmosis/app/params"
+)
+
+func GenerateGenesisCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export-genesis [input-snapshot-file]",
+		Short: "Export a genesis from fairdrop snapshot",
+		Long: `Export a genesis from fairdrop snapshot
```

Comment:
Perhaps `Export a genesis from fairdrop snapshot output by the airdrop command` ?

Reply:
Sunny wanted to do it as a different command as snapshot is published already.

Reply:
Agreed, but its not clear what a "snapshot" is or where it comes from if this isn't specified

Reply:
I see

Reply:
```suggestion
		Long: `Export a genesis from fairdrop snapshot output that was created by the airdrop command
```